public/views/ErrorPage.html

```diff
@@ -29,7 +29,7 @@
 				<div class="col-md-12 text-center">
 					<span id="errorTitle" class="display-1 d-block">Uh Oh!</span>
 					<div class="mb-4 lead"><strong>404 </strong>Something went wrong please check again later.</div>
-					<a href="#" class="btn btn-link">Return Home</a>
+					<a href="../home" class="btn btn-link">Return Home</a>
```

Comment:
Nitpick: `/home` as an absolute url path would probably be better in case the urls get shuffled around